### PR TITLE
Update shiro to 2.0.2 classifier jakarta

### PR DESCRIPTION
**Pull request by h2000**

### CODE REVIEW

**Comment by andi-huber** [comment]
Hi Alf, can you perhaps rebase your PR. I don't want to merge your changes to `AuthorizorShiro`.

**Comment by h2000** [comment]
Now only the pom is updated, when adding a classifier a version is needed too, but I'm not a maven expert ....

File: retired/shiro/security/shiro/pom.xml [hunk 1]
```diff
@@ -80,6 +80,8 @@
         <dependency>
             <groupId>org.apache.shiro</groupId>
             <artifactId>shiro-core</artifactId>
+            <classifier>jakarta</classifier>
+            <version>2.0.2</version>
```
Comment (lefou) [inline retired/shiro/security/shiro/pom.xml:84]: You should also update the `shiro.version` in `bom/pom.xml`.

```suggestion
            <version>${shiro.version}</version>
```

File: retired/shiro/security/shiro/pom.xml [hunk 2]
```diff
@@ -91,6 +93,8 @@
         <dependency>
             <groupId>org.apache.shiro</groupId>
             <artifactId>shiro-web</artifactId>
+            <classifier>jakarta</classifier>
+            <version>2.0.2</version>
```
Comment (lefou) [inline retired/shiro/security/shiro/pom.xml:97]:
```suggestion
            <version>${shiro.version}</version>
```

**Comment by lefou** [comment]
It could also be enough to add the classifier to the shiro dependencies in the `dependencyManagement` section in `bom/pom.xml`, but I didn't tested it.

**Comment by andi-huber** [comment]
I'll merge this PR. However, be aware: artifacts under the /retired/ folder are no longer part of the 3.x releases. And I don't think we'll reinstate those. But you can copy them to your own projects and maintain your own versions.

**Pull request merged**